### ISSUE TITLE
q13fox.com: nuisance

### ISSUE THREAD

**Issue by EW1S** [uBlockOrigin/uAssets#14547]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.q13fox.com/`

### Category

nuisance

### Describe the issue

I have blocked this over 200 times  and every time I go on this site,no matter how much I update things, no matter how many time I block it this banner comes up, every time I'm on the site and every time I go to an article.

You are seeing this message because ad or script blocking software is interfering with this page.
Disable any ad or script blocking software, then reload this page.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Bennerd](https://user-images.githubusercontent.com/25736361/187009738-85c164c3-274e-49e6-a279-e7f40098dace.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.0
Firefox: 102
filterset (summary): 
  network: 82099
  cosmetic: 44648
  scriptlet: 17697
  html: 729
listset (total-discarded, last updated): 
  added: 
    https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt: 0-0, 3h.59m
    https://mirror.cedia.org.ec/malwaredomains/justdomains: 0-0, never
  default: 
    user-filters: 438-65, never
    ublock-filters: 33181-33, 2d.3h.32m
    ublock-badware: 4218-1, 2d.3h.31m
    ublock-privacy: 247-0, 10d.20h.22m
    ublock-abuse: 76-0, 10d.20h.21m
    ublock-unbreak: 1851-29, 2d.3h.33m
    easylist: 68710-601, 2d.3h.30m
    easyprivacy: 27414-102, 2d.3h.29m
    plowe-0: 3630-451, 10d.20h.19m
    urlhaus-1: 6478-0, 1d.2h.36m
    ublock-quick-fixes: 389-4, 1d.2h.37m
filterset (user): [array of 439 redacted]
trustedset: 
  added: [array of 9 redacted]
  removed: 
    about-scheme
    edge-scheme
    moz-extension-scheme
switchRuleset: 
  added: [array of 8 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1474 ms (selfie)
  maxAssetCacheWait: 1075 ms
popupPanel: 
  blocked: 27
  blockedDetails: 
    amazon-adsystem.com: 1
    amplitude.com: 3
    audioeye.com: 1
    chartbeat.com: 2
    doubleclick.net: 1
    duckduckgo.com: 1
    fox.com: 1
    foxtv.com: 1
    go-mpulse.net: 1
    google.com: 5
    imasdk.googleapis.com: 1
    lura.live: 1
    mapbox.com: 4
    onelink.me: 1
    segment.com: 1
    taboola.com: 1
    webcontentassessor.com: 1
```
</details>

**Comment by MasterKia** [comment]
> - [X] I am not using another content blocker along uBO (also mind browser built-in blockers)

You're using Adblock Plus alongside uBlock Origin, please **remove it** and test again.

> - [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules

> user-filters: 438-65

Please test without custom filters ("My filters").

**Comment by Yuki2718** [comment]
> > * [x]  I am not using another content blocker along uBO (also mind browser built-in blockers)
> 
> You're using Adblock Plus and Privacy Badger alongside uBlock Origin, please **remove those** and test again (also disable your other add-ons just in case).
> 

This must be the cause, too many people fall into this completely wrong idea that runnint two blocker is somehow a good idea.

**Comment by MasterKia** [comment]
@Yuki2718 What is "GFC" anti-adb?
Searching didn't help.

**Comment by Yuki2718** [comment]
Google Funding Choice
Since it's too common we moved to generic fix below:
```
#@#.div-gpt-ad
*##.div-gpt-ad:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)
! for the french list
#@#div[class^="div-gpt-ad"]
#@#[class^="div-gpt-ad"]
*##[class^="div-gpt-ad"]:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)
#@#[id^="div-gpt-ad"]
*##[id^="div-gpt-ad"]:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)
#@#div[id^="div-gpt-"]
*##div[id^="div-gpt-"]:not([id^="div-gpt-ad"]):style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)
```
Using another blocker completely negates these filters by their cosmetic filter.

**Comment by MasterKia** [comment]
@Yuki2718 In that "uBOPa filter list" case, you mentioned gorhill **might be** considering to disallow this kind of generic procedural cosmetic filters (e.g. `##[class*="ads"]`).

So if he does that, it might affect this GFC anti-adb too?

**Comment by Yuki2718** [comment]
No, what may possibly be disabled is (highly) generic cosmetic filters but not specific-generic cosmetic filters used here. In fact it is [proposed](https://github.com/uBlockOrigin/uBlock-issues/discussions/1998) to convert really necessary generic cosmetic filters to specific-generic.

**Comment by EW1S** [comment]
I have no idea what the hell you are talking about.

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 06:35:56 AM GMT-8, Yuki2718  <> wrote:  
 
   ] 



No, what may possibly be disabled is (highly) generic cosmetic filters but not specific-generic cosmetic filters used here. In fact it is proposed to convert really necessary generic cosmetic filters to specific-generic.

**Comment by EW1S** [comment]
Not completed, not fixed.

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 02:21:59 AM GMT-8, Yuki2718  <> wrote:  
 
   ] 



Closed #14547 as completed.

**Comment by EW1S** [comment]
What the hell are you talking about?

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 02:22:03 AM GMT-8, Yuki2718  <> wrote:  
 
   ] 




   
   -  I am not using another content blocker along uBO (also mind browser built-in blockers)


You're using Adblock Plus and Privacy Badger alongside uBlock Origin, please remove those and test again (also disable your other add-ons just in case).


This must be the cause, too many people fall into this completely wrong idea that runnint two blocker is somehow a good idea.

**Comment by EW1S** [comment]
Absolutely no idea what this means.

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 02:50:39 AM GMT-8, Yuki2718  <> wrote:  
 
   ] 



Google Funding Choice
Since it's too common we moved to generic fix below:
#@#.div-gpt-ad*##.div-gpt-ad:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)! for the french list#@#div[class^="div-gpt-ad"]#@#[class^="div-gpt-ad"]*##[class^="div-gpt-ad"]:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)#@#[id^="div-gpt-ad"]*##[id^="div-gpt-ad"]:style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)#@#div[id^="div-gpt-"]*##div[id^="div-gpt-"]:not([id^="div-gpt-ad"]):style(width:1px!important;height:1px!important;min-width:1px!important;min-height:1px!important;margin:0!important;padding:0!important;overflow:hidden!important;opacity:0!important)
Using another blocker completely negates these filters by their cosmetic filter.

**Comment by MasterKia** [comment]
@EW1S Did you pay any attention to this?
👇🏻👇🏻👇🏻👇🏻
https://github.com/uBlockOrigin/uAssets/issues/14547#issuecomment-1229145819

**Comment by MasterKia** [comment]
> what may possibly be disabled is (highly) generic cosmetic filters but not specific-generic cosmetic filters used here

Got it. Also now I see that gorhill said [these filters](https://github.com/uBlockOrigin/uAssets/issues/14547#issuecomment-1229169938) won't be affected anyway since:
"_Where a HGCF is really the only sensible solution, we can use `*##...` to make this explicit as **these would not be discarded**._"

**Comment by EW1S** [comment]
This is completely useless, and I am the one with the issue, none of you are helping, and deleting what I am saying is further indication that none of you are interested in sorting this out. Thanks for absolutely NOTHING.

**Comment by MasterKia** [comment]
@EW1S As [I said here](https://github.com/uBlockOrigin/uAssets/issues/14546#issuecomment-1229218641) in plain English:
Please remove Adblock Plus (and optionally Privacy Badger) from your add-ons.

**Comment by EW1S** [comment]
AND AS I SAID IN PLAIN ENGLISH - USELESS! Condescending Jerk!

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 08:27:04 AM GMT-8, MasterKia  <> wrote:  
 
   ] 



@EW1S As I said here in plain English:
Please remove Adblock Plus (and optionally Privacy Badger) from your add-ons.

**Comment by MasterKia** [comment]
@EW1S We tried to reproduce your problem but we were unable to reproduce.

Your other problem reported here has been solved and is going to be fixed:
https://github.com/uBlockOrigin/uAssets/issues/14546

**Comment by Yuki2718** [comment]
@EW1S  We simply requested you to remove or disable Adblock Plus which is the cause of your problem. Don't you get it?

**Comment by EW1S** [comment]
I DID IT DID NOT CHANGE ANYTHING - DON"T YOU GET IT?!

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 09:03:37 AM GMT-8, Yuki2718  <> wrote:  
 
   ] 



@EW1S We simply requested you to remove or disable Adblock Plus which is the cause of your problem. Don't you get it?

**Comment by Yuki2718** [comment]
I haven't heard that.

**Comment by Yuki2718** [comment]
Did you also disable Privacy badger?

**Comment by Yuki2718** [comment]
Please create a new browser profile, fresh install uBO, and test again.

**Comment by Infrastructural** [comment]
@ryanbr `q13fox.com##.ad-container`

**Comment by EW1S** [comment]
Don't know what to tell you other than uninstalling ABP and Privacy Badger did NOT fix the problem. Just because you cannpt replicate it does not mean it does not exist, and in the future, don't insult people when they tell you they already did what you asked them to do. It makes you look bad.

CONFIDENTIALITY NOTICE: This communication is intended for the sole use of the individual and entity to whom it is addressed, and may contain information that is privileged, confidential and exempt from disclosure under applicable law. You are hereby notified that any dissemination, distribution or duplication of this communication by someone other than the intended addressee or its designated agent is strictly prohibited. DO NOT FORWARD.


     On Saturday, August 27, 2022, 08:34:08 AM GMT-8, MasterKia  <> wrote:  
 
   ] 



@EW1S We tried to reproduce your problem but we were unable to reproduce.

Your other problem reported here has been solved and is going to be fixed:
#14546
